docker-compose.yml

```diff
@@ -28,6 +28,11 @@ services:
     volumes:
       - fsdata:/data
 
+  sass:
```

Comment:
If this is going to merged, I'd suggest making these changes to `docker-compose.override.yml` instead. Developers running the Python code locally may want more control to run the SASS watcher locally too.

Reply:
Sounds good to me, thanks Brian!